test/e2e/config/vsphere-ci.yaml

```diff
@@ -60,10 +60,17 @@ providers:
     files:
     # Add a cluster template
     - sourcePath: "../../../test/e2e/data/infrastructure-vsphere/main/cluster-template.yaml"
+    - sourcePath: "../../../test/e2e/data/infrastructure-vsphere/main/cluster-template-install-on-bootstrap.yaml"
+    - sourcePath: "../../../test/e2e/data/infrastructure-vsphere/main/clusterclass-quick-start.yaml"
```

Comment:
Why do we need this quick-start template?

Reply:
This is because `install-on-boootstrap` is leveraging a clusterclass json patch to replace the node's kubernetes components version, so we have to make the test workload cluster be a clusterclass based cluster